server/test/test-server.js

```diff
@@ -34,31 +37,31 @@ describe(`Page load test for "/" endpoint`, function() {
 	})
 })
 
-describe(`Test for "/signin" endpoint`, function() {
-	it('Should return 400 error with invalid user', done => {
+describe(`Test for "/signin" endpoint`, function () {
+	it('Should return 400 error with invalid user', (done) => {
 		chai
 			.request(server)
 			.post('/signin')
 			.send({
 				username: 'test2',
-				password: 'test2'
+				password: 'test2',
 			})
-			.end(function(err, res) {
+			.end(function (err, res) {
 				expect(err).to.be.a('null')
 				res.should.have.status(400)
 				done()
 			})
 	})
 
-	it('Should be able to login with test account', done => {
+	it('Should be able to login with test account', (done) => {
 		chai
 			.request(server)
 			.post('/signin')
 			.send({
 				username: 'test',
-				password: 'test'
+				password: 'test',
 			})
-			.end(function(err, res) {
+			.end(function (err, res) {
 				// expect(err).to.be.a('null')
```

Comment:
I'm not sure if this test should still be commented out

Reply:
it shouldn't be, but I'm not sure how to solve this just yet